{
  "containers": {
    "cna": {
      "providerMetadata": {
        "orgId": "f0158376-9dc2-43b6-827c-5f631a4d8d09"
      },
      "title": "Improper authorization bypass on row level security via SQL Injection",
      "problemTypes": [
        {
          "descriptions": [
            {
              "description": "CWE-89 Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection')",
              "lang": "en",
              "cweId": "CWE-89",
              "type": "CWE"
            }
          ]
        }
      ],
      "source": {
        "discovery": "UNKNOWN"
      },
      "affected": [
        {
          "vendor": "Apache Software Foundation",
          "product": "Apache Superset",
          "collectionURL": "https://pypi.python.org",
          "versions": [
            {
              "status": "affected",
              "version": "0",
              "lessThan": "4.1.2",
              "versionType": "semver"
            }
          ],
          "defaultStatus": "unaffected"
        }
      ],
      "descriptions": [
        {
          "value": "An authenticated malicious actor using specially crafted requests could bypass row level security configuration by injecting SQL into 'sqlExpression' fields. This allowed the execution of sub-queries to evade parsing defenses ultimately granting unauthorized access to data.\n\nThis issue affects Apache Superset: before 4.1.2.\n\nUsers are recommended to upgrade to version 4.1.2, which fixes the issue.",
          "lang": "en",
          "supportingMedia": [
            {
              "type": "text/html",
              "base64": false,
              "value": "<p>An authenticated malicious actor using specially crafted requests could bypass row level security configuration by injecting SQL into 'sqlExpression' fields. This allowed the execution of sub-queries to evade parsing defenses ultimately granting unauthorized access to data.</p><p>This issue affects Apache Superset: before 4.1.2.</p><p>Users are recommended to upgrade to version 4.1.2, which fixes the issue.</p>"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://lists.apache.org/thread/ms2t2oq218hb7l628trsogo4fj7h1135",
          "tags": [
            "vendor-advisory"
          ]
        }
      ],
      "metrics": [
        {
          "format": "CVSS",
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ],
          "cvssV4_0": {
            "version": "4.0",
            "attackVector": "NETWORK",
            "attackComplexity": "LOW",
            "attackRequirements": "NONE",
            "privilegesRequired": "LOW",
            "userInteraction": "NONE",
            "vulnConfidentialityImpact": "HIGH",
            "subConfidentialityImpact": "NONE",
            "vulnIntegrityImpact": "NONE",
            "subIntegrityImpact": "NONE",
            "vulnAvailabilityImpact": "NONE",
            "subAvailabilityImpact": "NONE",
            "Safety": "NOT_DEFINED",
            "Automatable": "NOT_DEFINED",
            "Recovery": "NOT_DEFINED",
            "valueDensity": "NOT_DEFINED",
            "vulnerabilityResponseEffort": "NOT_DEFINED",
            "providerUrgency": "NOT_DEFINED",
            "baseSeverity": "HIGH",
            "baseScore": 7.1,
            "vectorString": "CVSS:4.0/AV:N/AC:L/AT:N/PR:L/UI:N/VC:H/VI:N/VA:N/SC:N/SI:N/SA:N"
          }
        }
      ],
      "credits": [
        {
          "lang": "en",
          "value": "Pedro Sousa",
          "type": "coordinator"
        },
        {
          "lang": "en",
          "value": "Beto de Almeida",
          "type": "remediation developer"
        },
        {
          "lang": "en",
          "value": "Mirakl Security",
          "type": "finder"
        }
      ],
      "x_generator": {
        "engine": "Vulnogram 0.2.0"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2025-48912",
    "assignerOrgId": "f0158376-9dc2-43b6-827c-5f631a4d8d09",
    "serial": 1,
    "state": "PUBLISHED"
  },
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1"
}